types/types.go

```diff
@@ -341,6 +341,11 @@ type Type struct {
 	// If Kind == func, this is the signature of the function.
 	Signature *Signature
 
+	// Constval contains a stringified constant value if
+	// Kind == DeclarationOf and this is a constant value
+	// declaration.
+	Constval *string
```

Comment:
Can we go with `ConstValue` for the name?

Reply:
Fixed.